packages/api/src/routes/medical/organization.ts

```diff
@@ -83,9 +85,14 @@ router.get(
   requestLogger,
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
+    const output = getOutputFormatFromRequest(req);
 
     const org = await getOrganization({ cxId });
-    return res.status(status.OK).json(org ? dtoFromModel(org) : undefined);
+
+    const respStatus = status.OK;
+    if (!org) return res.status(respStatus).json(undefined);
+    if (output === "fhir") return res.status(respStatus).json(toFHIR(org));
```

Comment:
Is this something customers requested..?

Reply:
ditto about this. whats the context + do we need to update docs?

Reply:
internal usage, we can discuss whether we want to expose it officially, but useful for debugging purposes